test/integration/etcd/data.go

```diff
@@ -226,6 +226,16 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		// --
 
 		// k8s.io/kubernetes/pkg/apis/networking/v1
+		gvr("networking.k8s.io", "v1", "ingresses"): {
+			Stub:             `{"metadata": {"name": "ingress3"}, "spec": {"defaultBackend": {"serviceName": "service", "servicePort": 5000}}}`,
```

Comment:
`{"serviceName": "service", "servicePort": 5000}` fields do not exist in v1... 

this should be `{"service":{"name":"service", "port":{"number": 5000}}`, right?

this reveals a validation gap that allows backends that do not specify `service` or `resource`

Reply:
Oops, will change and add that extra validation